src/modules/sensors/sensors.cpp

```diff
@@ -2070,6 +2070,11 @@ Sensors::task_main()
 	/* start individual sensors */
 	int ret = 0;
 
+/*
```

Comment:
Good point.  The DF stub class I created should solve the problem, but we need to get DF to successfully build with the QURT config.